### PR TITLE
{bio}[foss/2018b,fosscuda/2018b] GROMACS v2018.4 with PLUMED 2.5.0

### PR DESCRIPTION
**Pull request by jhein32**
This is an update on the still pending PR #7143.  PLUMED 2.5.0 is now released (the former PR uses a beta version).  PLUMED 2.5.0 supports the newer GROMACS 2018.4.   

I assume this PR will also suffer from EB not tolerating multiple PLUMED versions in foss/2018b and fosscuda/2018b.  This should be fixed.

### CODE REVIEW

**Comment by akesandgren** [comment]
@boegel We need an exception for PLUMED. Packages that use PLUMED are very version specific.

**Comment by ocaisa** [comment]
You can modify the test at/near https://github.com/easybuilders/easybuild-easyconfigs/blob/master/test/easyconfigs/easyconfigs.py#L200 to account for this

**Comment by akesandgren** [comment]
Yeah, I know. I was just raising the issue to give others time to give input.

**Comment by jhein32** [comment]
@ocaisa @akesandgren @boegel  
I agree, that adding the modification of easyconfigs.py to this PR makes every sense in the world.  However, it would take me some time to understand the code in question (easyconfigs.py) and to propose a modification.   

Any ideas on how to progress this?  I allowed for "edits from maintainers".

**Comment by jhein32** [comment]
Just for reference: https://plumed.github.io/doc-v2.5/user-doc/html/_installation.html#Patching says that PLUMED 2.5.0 can work with GROMACS 2016.5 and 2018.4.

**Comment by ocaisa** [comment]
I don't know that code that well, but I think if you just change the list in that line to `['Boost', 'PLUMED', 'R']` the test should pass

**Comment by boegel** [comment]
@jhein32 The exception you added will work if you also include the `PLUMED` version in the `versionsuffix` (like we do for `Python`)

**Comment by akesandgren** [comment]
@jhein32 Can you please change it to include the PLUMED version as @boegel says.
It's something we should have done from the beginning.

**Comment by jhein32** [comment]
I will have a look and get back.

File: easybuild/easyconfigs/p/PLUMED/PLUMED-2.5.0-foss-2018b-Python-2.7.15.eb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+preconfigopts = 'env FC=$MPIF90 LIBS="$LIBLAPACK $LIBS" '
+configopts = ' --exec-prefix=%(installdir)s --enable-gsl --enable-modules=all --enable-python'
+prebuildopts = 'source sourceme.sh && '
+# install path for PLUMED libraries must be included in $LD_LIBRARY_PATH when Python bindings get built/installed
+preinstallopts = 'LD_LIBRARY_PATH="%(installdir)s/lib:$LD_LIBRARY_PATH" '
+
+sanity_check_paths = {
+    'files': ['bin/plumed', 'lib/libplumedKernel.%s' % SHLIB_EXT, 'lib/libplumed.%s' % SHLIB_EXT,
+              'lib/plumed/python/plumed.%s' % SHLIB_EXT],
+    'dirs': ['lib/plumed'],
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/PLUMED/PLUMED-2.5.0-foss-2018b-Python-2.7.15.eb]: Redundant due to the checks in "files", drop it.

Comment (akesandgren) [reply easybuild/easyconfigs/p/PLUMED/PLUMED-2.5.0-foss-2018b-Python-2.7.15.eb]: same in the other plumed .eb

Comment (jhein32) [reply easybuild/easyconfigs/p/PLUMED/PLUMED-2.5.0-foss-2018b-Python-2.7.15.eb]: Åke, not fully sure what you are after here.   You want me to drop the 

```
'dirs': ['lib/plumed'],
```

line?  Please confirm or explain better.

Ta
   Joachim

Comment (akesandgren) [reply easybuild/easyconfigs/p/PLUMED/PLUMED-2.5.0-foss-2018b-Python-2.7.15.eb]: Change to
'dirs': []

**Comment by akesandgren** [comment]
@jhein32 Can you produce a test report too please?
eb --from-pr=7478 -r -u --rebuild (to make sure it rebuilds all the .eb's in it)
I'll make one too but it will be a while before i can do it.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/9736893bb8e602b69ae40ec8d2f2fd05 for a full test report.

**Comment by akesandgren** [comment]
Unless users have already started using it there should be no problem, it will perform the same operation as you do when building it yourself.
But now i have a test report from our system. should be enough.

**Comment by akesandgren** [comment]
Going in, thanks @jhein32!

**Pull request merged**